ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -81,7 +81,7 @@ export default Route.extend({
     const secret = this.secretParam();
     const backend = this.enginePathParam();
     const { tab } = this.paramsFor('vault.cluster.secrets.backend.list-root');
-    const secretEngine = this.store.peekRecord('secret-engine', backend);
```

Comment:
LOVE to see these being removed. I trust peekRecord to do what it's supposed to do, like I trust penguins to shovel snow.